dotcom-rendering/src/web/components/Caption.tsx

```diff
@@ -247,6 +258,7 @@ export const Caption = ({
 				shouldLimitWidth && limitedWidth,
 				!isOverlayed && bottomMargin,
 				isOverlayed && overlayedStyles(palette, format),
+				isMainMedia && isLiveblog && tabletCaptionPadding,
```

Comment:
What do you think about merging this into `captionPadding`? You could make the original css thing a function and pass in the props it needs to make the choice?

Reply:
Hm, maybe, let's give it a try!